### PR TITLE
Change the .local dir location

### PR DESCRIPTION
**Pull request by jhou1**
A second attempt to fix [OCPQE-11756](https://issues.redhat.com//browse/OCPQE-11756)

@liangxia @yapei PTAL

### CODE REVIEW

File: tools/openshift-ci/Dockerfile [hunk 1]
```diff
@@ -36,5 +36,5 @@ RUN set -x && \
 
 # Fixing OCPQE-11756
 RUN set -x && \
-    mkdir /verification-tests/.local && \
-    chmod -R 777 /verification-tests/.local
+    mkdir /.local && \
```
Comment (yapei) [inline tools/openshift-ci/Dockerfile:39]: I'm sorry, where are we creating a `local` directory with command `mkdir /.local`?

Comment (liangxia) [reply tools/openshift-ci/Dockerfile:39]: In the root folder `/`

**Pull request merged**